cmd/lncli/commands.go

```diff
@@ -3251,6 +3251,7 @@ var updateChannelPolicyCommand = cli.Command{
 			Name: "time_lock_delta",
 			Usage: "the CLTV delta that will be applied to all " +
 				"forwarded HTLCs",
+			Value: 144,
```

Comment:
Would rather have this set as a constant in the `routing` package. Otherwise, we'll need to always change this value in _multiple_ places.